examples/22_site_doe/driver_config.yaml

```diff
@@ -0,0 +1,40 @@
+description: Driver config for CSVGen design of experiments
+name: driver_config
+
+general:
+  folder_output: ex_22_out
+  create_om_reports: false
+driver:
+  design_of_experiments:
+    flag: true
+    debug_print: true
+    generator: "csvgen"
+    filename: "site_solar_doe.csv"
+    run_parallel: false
+design_variables:
+  solar:
+    capacity_kWdc:
+      flag: true
+      lower: 5000.0
+      upper: 5000000.0
```

Comment:
Non-blocking: 5GW is so big! It doesn't matter here, but in case a user looks at this, maybe have a lower upper bound? disregard if this his on purpose